news/2018-05-21-project-loved-week-of-may-20th.md

```diff
@@ -0,0 +1,242 @@
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/taiko/chousai-kenbo-sengen.jpg)](https://osu.ppy.sh/community/forums/topics/749064)
+
+[Hommarju feat. R. Cena - Chousai Kenbo Sengen](https://osu.ppy.sh/beatmapsets/689734) by [iceOC](https://osu.ppy.sh/users/5482401) and [Gezoda](https://osu.ppy.sh/users/481582)  
+*written by [Gezoda](https://osu.ppy.sh/users/481582)*
+
+Technical map with a pinch of catgirls, Chousai Kenbo Sengen really is something you should give a try. Mapped by iceOC, an obscure Japanese mapper, this tech beatmap comes with loads of 1/6ths, handswitches as well as streams and comes with a whopping HP7 for 2,000 notes, meaning that clearing it will be a challenge. But if you're too scared of this, an easier Oni difficulty mapped by Gezoda is provided as well, but is still going to test your stamina.
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/taiko/hachigatsu-no-yoru.jpg)](https://osu.ppy.sh/community/forums/topics/749065)
+
+[Silent Siren - Hachigatsu no Yoru](https://osu.ppy.sh/beatmapsets/410556) by [suni](https://osu.ppy.sh/users/4126437)  
+*written by [Backfire](https://osu.ppy.sh/users/263110)*
+
+With over 30 favorites and a great grasp of rhythm and just the right amount of challenge to keep most players entertained, you would figure this wasn't [suni](https://osu.ppy.sh/users/4126437)'s first map but, you'd be surprised to find out that it is! For a first map, [Hachigatsu no Yoru](https://osu.ppy.sh/beatmapsets/410556) is certainly impressive. With its fun variety of patterns and infectiously catchy use of vocal mapping, you really can't go wrong!
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/taiko/the-medley-of-kirby-ssdx.jpg)](https://osu.ppy.sh/community/forums/topics/749066)
+
+[Hapi - THE MEDLEY OF KIRBY SSDX](https://osu.ppy.sh/beatmapsets/205717) by [alecreit](https://osu.ppy.sh/users/2586173) et al.  
+*written by [Backfire](https://osu.ppy.sh/users/263110)*
+
+This one was for the ages. Featuring a list of over 30 incredible and noteworthy Japanese mappers, this is their biggest and best collaboration. A 15 minute medley of Kirby music and an epic collaboration of mappers can only equal pure unadulterated joy over the length of your play. Between the storyboard and the music, this is one huge nostalgia trip for Taiko nerds all around the world. Names that you may recognize, names you may never have seen before, one cannot help but feel happy for the joy that Taiko brings to the Japanese community, and we all get to celebrate it with this map!
```

Comment:
Lowercase `Taiko` to be consistant. If it's mentioning the game mode directly, use `osu!taiko`.